core/src/main/java/com/distkv/core/block/BlockPool.java

```diff
@@ -0,0 +1,126 @@
+package com.distkv.core.block;
+
+import sun.misc.Unsafe;
+
+import java.io.Closeable;
+import java.lang.reflect.Field;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class BlockPool implements Closeable {
+
+  private static Unsafe getUnsafe() {
+    try {
+      Field field = Unsafe.class.getDeclaredField("theUnsafe");
+      field.setAccessible(true);
+      return (Unsafe) field.get(null);
+    } catch (Exception e) {
```

Comment:
`Exception` is too broad. Better to use the concrete exception type here.

Reply:
i will modify for this.